### PR TITLE
PS-531 workflow

### PR DESCRIPTION
**Pull request by 4rthem**

### CODE REVIEW

File: databox/api/src/Asset/Attribute/InitialAttributeValuesResolver.php [hunk 1]
```diff
@@ -131,7 +132,11 @@ private function resolveInitial(
         // remove empty values
         return array_filter(
             $initialValues,
-            function (string $s): bool {
+            function (?string $s): bool {
+                if (null === $s) {
```
Comment (jygaulier) [inline databox/api/src/Asset/Attribute/InitialAttributeValuesResolver.php:136]: if we return "" as value for blank tag, we _may_ not need to test null.
But let's keep this test to protect `trim()` not accepting null anymore.

Comment (jygaulier) [inline databox/api/src/Asset/Attribute/InitialAttributeValuesResolver.php:136]: if we return "" as value for blank tag, we _may_ not need to test null.
But let's keep this test to protect `trim()` not accepting null anymore.

File: databox/api/src/File/FileMetadataAccessorWrapper.php [hunk 2]
```diff
@@ -58,7 +58,7 @@ public function getMetadata(string $id)
 
         // no file or no metadata ? be nice also
         $r = MetadataNormalizer::createBlankMeta();
-        $this->meta[$id]['value'] = null;
+        $this->meta[$id] = $r;
```
Comment (jygaulier) [inline databox/api/src/File/FileMetadataAccessorWrapper.php:61]: YES

File: databox/api/src/Metadata/MetadataNormalizer.php [hunk 3]
```diff
@@ -47,6 +47,7 @@ public static function createBlankMeta(): array
         return [
             'name' => null,
             'values' => [],
+            'value' => null,
```
Comment (jygaulier) [inline databox/api/src/Metadata/MetadataNormalizer.php:50]: should be "" to protect twig, but anyway empty value(s) are **now** removed before render.
todo (?): filtering empty value(s) may prevent to initialize an attribute with **real** empty value from metadata.
So let's keep null as discriminant, and fix filtering to preserve real empty string.

File: databox/api/src/Asset/Attribute/InitialAttributeValuesResolver.php (hunk 1 above)
Comment (4rthem) [reply databox/api/src/Asset/Attribute/InitialAttributeValuesResolver.php:136]: `trim` does not accept null values in PHP8.
Indeed you need to review value normalization, with unit tests :)
In your next PR?

File: databox/api/src/Metadata/MetadataNormalizer.php (hunk 3 above)
Comment (4rthem) [reply databox/api/src/Metadata/MetadataNormalizer.php:50]: Need to be revisited in general, cf my previous comment
